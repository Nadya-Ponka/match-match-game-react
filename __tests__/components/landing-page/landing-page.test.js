import React from 'react';
import { shallow } from 'enzyme';

import LandingPage from '../../../src/components/landing-page/landing-page';

describe('LandingPage', () => {
    test('match snapshot', () => {
        const wrapper = shallow(<LandingPage />);
        expect(wrapper).toMatchSnapshot();
    });
});
