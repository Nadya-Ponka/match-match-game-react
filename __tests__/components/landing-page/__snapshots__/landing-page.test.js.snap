// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`LandingPage match snapshot 1`] = `
<div
  className="rules"
>
  <img
    alt=""
    className="title"
    src="../../../Images/title.svg"
  />
  <p>
    Давай найдем каждому Смешарику пару! Переворачивай любые две карточки. Если они одинаковые — пара найдена. Найди пары для всех. И поспеши!
  </p>
  <Button
    className="buttonStart"
    onClick={[Function]}
    text="Начинай"
  />
</div>
`;
